server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/GetMessageListMethodTest.java

```diff
@@ -608,4 +609,8 @@ public void getMessageListShouldChainFetchingMessagesWhenAskedFor() throws Excep
             .body("[1][1].list", hasSize(1))
             .body("[1][1].list[0].id", equalTo(message.getMessageId().serialize()));
     }
+
+    private Date convertToDate(LocalDate localDate) {
+        return Date.from(localDate.atStartOfDay(ZoneId.systemDefault()).toInstant());
```

Comment:
never use systemDefault, it will have different result on different system, testing is about reproducibility